src/Goerli-DssSpell.sol

```diff
@@ -186,22 +318,126 @@ contract DssSpellAction is DssAction {
 
         // RWA-010 Onboarding
         // https://vote.makerdao.com/polling/QmNucsGt
-        // TODO
+        _addCentrifugeCollateral(CentrifugeCollateralValues({
+            GEM_JOIN:        MCD_JOIN_RWA010_A,
+            GEM:             RWA010,
+            URN:             RWA010_A_URN,
+            OPERATOR:        RWA010_A_OPERATOR,
+            INPUT_CONDUIT:   RWA010_A_INPUT_CONDUIT,
+            OUTPUT_CONDUIT:  RWA010_A_OUTPUT_CONDUIT,
+            DROP:            0xd7943e68bD284dAd75A59d07Fab7708a21B8a95E,
+            OWNER:           0xa78F096D4cfc32637513e02Ddf020EFb3fFf4df1, // Tinlake Clerk
+            POOL:            0xd7BD4F27302aBDB8292F534BF52e7d10dDf6A112, // Tinlake Operator
+            TRANCHE:         0xe12c278c7e6c8B64E322d9cce66E2C9177051FeD, // Tinlake Tranche
+            gemID:           "RWA010",
+            joinID:          "MCD_JOIN_RWA010_A",
+            urnID:           "RWA010_A_URN",
+            inputConduitID:  "RWA010_A_INPUT_CONDUIT",
+            outputConduitID: "RWA010_A_OUTPUT_CONDUIT",
+            pipID:           "PIP_RWA010",
+            ilk:             "RWA010-A",
+            ilkString:       "RWA010-A",
```

Comment:
Should be `RWA010`.

Reply:
All of the other collateral types follow `RWAXXX-A` convention.

Reply:
Oh, for ilkString, yes.